IdentityCore/src/logger/MSIDLogger+Internal.h

```diff
@@ -88,6 +91,8 @@ MSID_LOG(MSIDLogLevelVerbose, nil, _ctx, YES, _fmt, ##__VA_ARGS__)
 #define MSID_LOG_VERBOSE_CORR_PII(_correlationId, _fmt, ...) \
 MSID_LOG(MSIDLogLevelVerbose, _correlationId, nil, YES, _fmt, ##__VA_ARGS__)
 
+#define MSID_TRACE /**/
```

Comment:
`// Unused`
Comments are not included in macros.

Reply:
changed